### ISSUE TITLE
Progressive Jpeg in not loading when using a Post Processor with Request

### ISSUE THREAD

**Issue by AmniX** [facebook/fresco#1637]
Not able to process Progressive Image with Post Process, Its not settings on ImageView At All. Is there any Solution to do the same? 
I really want to replicate [this](https://github.com/ibireme/YYWebImage) iOS Library with Fresco as Fresco is the only way to Load Progressive Jpeg in Android for now.

**Comment by oprisnik** [comment]
Yeah, I think we currently don't support postprocessors for animated images and progessive JPEGs

**Comment by omainegra** [comment]
Any progress on this? I'd expect the post-processor is applied to every image of the progressive JPG, any workaround? Thank you
